openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -14,7 +14,6 @@
     WorkfileBuildPlaceholderDialog,
 )
 
-from .lib import read, imprint
```

Comment:
Where is this PR creating a circular/cyclic import even though the only import changing is this one? I don't see any imports being added and thus that means any cycling imports must have already existed prior to this PR; right?

As far as I know you can revert _this_ change if you only keep the `from .workfile_template_builder import build_workfile_template` import as you have refactored in the function in lib. Right?

Reply:
I don't think this moved import is still necessary, right? Or is it?